boost-maven-plugin/src/main/java/boost/project/LibertyBoostMojo.java

```diff
@@ -91,16 +102,33 @@ public void execute() throws MojoExecutionException {
 			} else {
 				// dealing with an EE based app
 				installApp("project");
-				boosterFeatures = getBoosterConfigsFromDependencies(mavenProject);
+				boosterFeatures = getBoosterConfigsFromDependencies(project);
 				generateServerXMLJ2EE(boosterFeatures);
 			}
         } catch ( TransformerException | ParserConfigurationException e) {
             throw new MojoExecutionException("Unable to generate server configuration for the Liberty server", e);
         }
 
         installMissingFeatures();
+        
+        // Copy the Spring Boot uber JAR back as the project artifact
+        try {
+            File springJar = new File(springBootUtil.getSpringBootUberJarPath(project.getArtifact().getFile()));
+            if(springJar.exists()) {
+                FileUtils.copyFile(springJar, project.getArtifact().getFile());
```

Comment:
As we discussed, this copy is not always necessary. If we just copied the SB uber JAR over at line 87, we do not need to copy it back. That utility method would have to return a boolean indicating that the copy occurred though.

Reply:
Opened https://github.com/OpenLiberty/boost-common/pull/8 to fix this.